src/TeachMeSkills/Homework/Homework2.java

```diff
@@ -0,0 +1,142 @@
+package TeachMeSkills.Homework;
+import java.util.Scanner;
+
+public class Homework2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        // Управляющие конструкции. Задание 1
+        System.out.print("Введите номер месяца, чтобы узнать пору года: ");
+        String season;
+        int month = sc.nextInt();
+        season = switch (month) {
+            case 12, 1, 2 -> "Зима";
+            case 3, 4, 5 -> "Весна";
+            case 6, 7, 8 -> "Лето";
+            case 9, 10, 11 -> "Осень";
+            default -> "Введённое число не является номером месяца";
+        };
+        System.out.println(season);
+
+        // Управляющие конструкции. Задание 2
+        System.out.print("Введите номер месяца, чтобы узнать пору года: ");
+        month = sc.nextInt();
+        if (month == 12 || month == 1 || month == 2) {
+            season = "Зима";
+        } else if (month == 3 || month == 4 || month == 5) {
+            season = "Весна";
+        } else if (month == 6 || month == 7 || month == 8) {
+            season = "Лето";
+        } else if (month == 9 || month == 10 || month == 11) {
+            season = "Осень";
+        } else {
+            season = "Введённое число не является номером месяца";
+        }
+        System.out.println(season);
+
+        // Управляющие конструкции. Задание 3
+        System.out.print("Введите число: ");
+        int number = sc.nextInt();
+        if (number % 2 == 0) {
+            System.out.println("Введённое число чётное");
+        } else {
+            System.out.println("Введённое число нечётное");
+        }
+
+        // Управляющие конструкции. Задание 4
+        System.out.print("Введите значение температуры: ");
+        int temperature = sc.nextInt();
+        if (temperature > -5){
+            System.out.println("Тепло");
+        } else if(temperature > -20){
+            System.out.println("Нормально");
+        } else {
+            System.out.println("Холодно");
+        }
+
+        // Управляющие конструкции. Задание 5
+        System.out.print("Введите номер цвета радуги: ");
+        String color;
+        int colorNumber = sc.nextInt();
+        color = switch (colorNumber) {
+            case 1 -> "Красный";
+            case 2 -> "Оранжевый";
+            case 3 -> "Жёлтый";
+            case 4 -> "Зелёный";
+            case 5 -> "Голубой";
+            case 6 -> "Синий";
+            case 7 -> "Фиолетовый";
+            default -> "Введённое число не является номером цвета радуги";
+        };
+        System.out.println(color);
+
+        // Циклы. Задание 1
+        for(number = 0; number < 100; number++) {
+            if (number % 2 != 0) {
+                System.out.print(" " + number);
+            }
+        }
+
+        // Циклы. Задание 2
+        number = sc.nextInt();
+        int result = 0;
+        if (number >= 0) {
+            for (int sum = 2; sum < number; sum++) {
```

Comment:
Is it correct?

Reply:
Yes, if we exclude 1 and the entered number

Reply:
But why did you exclude 1?

Reply:
Not fixed

Reply:
fixed